CMakeLists.txt

```diff
@@ -419,7 +419,6 @@ endif()
 
 # Configuring other platform dependencies
 if(APPLE)
-  set(CMAKE_OSX_DEPLOYMENT_TARGET "10.10")
```

Comment:
Due to this we see warnings when building on newer versions of macOS:
<img width="1785" alt="Screenshot 2022-07-23 at 16 32 41" src="https://user-images.githubusercontent.com/266271/180609579-55e3c19f-1441-49bd-8903-13bc5c6802a6.png">

I don't see any reason to have it tbh.

Reply:
Hm, this was added by the reason https://github.com/Project-OSRM/osrm-backend/pull/3198

Not sure it is still relevant, but removing this change for the time being.